x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/last_value.tsx

```diff
@@ -22,9 +22,17 @@ function ofName(name: string) {
 
 const supportedTypes = new Set(['string', 'boolean', 'number', 'ip']);
 
-export function sortFieldIsInvalid(sortField: string, indexPattern: IndexPattern) {
+export function getInvalidSortFieldMessage(sortField: string, indexPattern?: IndexPattern) {
+  if (!indexPattern) {
+    return;
+  }
   const field = indexPattern.getFieldByName(sortField);
-  return !(field && field.type === 'date');
+  if (!(field && field.type === 'date')) {
+    return i18n.translate('xpack.lens.indexPattern.lastValue.invalidSortField', {
+      defaultMessage: 'Field {invalidField} has an invalid reference',
```

Comment:
nit: I know it didn't get introduced on this PR, but the wording seems super technical - maybe
* "Field {invalidField} was not found"
* "Field {invalidField} is not a date field and can't be used for sorting"?

We can also do a pass over all wording at once in a separate PR